scripts/start.sh

```diff
@@ -1,9 +1,21 @@
 #!/bin/bash -e
 
-export ANDROID_API_FOR_ABI_32=24
-export ANDROID_API_FOR_ABI_64=24
+export ANDROID_API_FOR_ABI_32=29
+export ANDROID_API_FOR_ABI_64=29
+export ANDROID_TARGET_API=35
 export ROOTDIR=$PWD
 
+# Default toolchain locations when not provided.
+DEFAULT_ANDROID_HOME="$HOME/Library/Android/sdk"
+if [[ -z "$ANDROID_HOME" || ! -d "$ANDROID_HOME" ]]; then
+  export ANDROID_HOME="$DEFAULT_ANDROID_HOME"
+fi
+
+DEFAULT_ANDROID_NDK="$ANDROID_HOME/ndk/28.2.13676358"
```

Comment:
```suggestion
DEFAULT_ANDROID_NDK="$ANDROID_HOME/ndk/27.1.12297006"
```

maybe align the ndk version with react-native

Reply:
The clang in NDK 27.1 is known to miscompile the computed gotos, which we run into in newer interpreters: https://github.com/android/ndk/issues/2040

I'd at least suggest bumping to r27d (27.3.13750724). note that this bug may not affect some platform holders who effectively roll their own NDK with an internally-maintained LLVM/clang distro.

that said, in NDK 28, libc++ now includes debug info, which I've found invaluable when debugging in simulator and on-device.

Reply:
we now generate a pacakge for NDK 27, 28, and 29 so that people can choose whichever is appropriate for their deployment context.